src/routes/JobCreateRoute.js

```diff
@@ -9,7 +9,16 @@ class JobCreateRoute extends React.Component {
   static manifest = Object.freeze({
     jobs: {
       type: 'okapi',
-      path: 'erm/jobs/packageImport',
+      path: (_q, _p, _r, _l, props) => {
+        const { match: { path } } = props;
+        let endpointPath;
+        if (path === '/local-kb-admin/create/KBART') {
+          endpointPath = 'erm/jobs/kbartImport';
+        } else {
+          endpointPath = 'erm/jobs/packageImport';
+        }
+        return endpointPath;
```

Comment:
This approach is fine for now but I am would like to know what @doytch thinks if we should adopt this approach going forward vs having one route at the top level `index.js` like
`${path}/create/:format(KBART|JSON)` 
and using the `params` to construct the path?

The 2 routes approach although look redundant seems to be more understandable to the user, but the one route approach looks much cleaner I feel, and then there is also the case of extensibility. 

Also, I am not sure if we could do something like `${path}/create/:format(KBART=kbartImport|JSON=packageImport)`

and then we wouldn't to define `path` as a function, and just have the
`path='/local-kb-admin/create/%format'`

Thats something that needs more research, and I am just leaving it as an option here

Reply:
Having spoken to Mark, this is now done :)